hdl-tests/shouldwork/Xilinx/Ibufds.hs

```diff
@@ -0,0 +1,30 @@
+module Ibufds where
+
+import Clash.Explicit.Prelude
+
+import Clash.Cores.Xilinx.Ibufds
+import Clash.Explicit.Testbench
+import Clash.Xilinx.ClockGen
+
+type Dom = XilinxSystem
+
+topEntity ::
+  DiffClock Dom ->
+  Signal Dom (Index 10)
+topEntity diffClk = o
+ where
+  clk = ibufds diffClk
+  o = register clk resetGen enableGen 0 $ fmap (satSucc SatBound) o
```

Comment:
Maybe I'm being overly critical, but the `resetGen` really caught my attention. It will hold a reset during the time that the `DiffClock Dom` needs to be reset. So the first thing that occurred to me was: is there a delay inside `ibufds`? Does this reset even reach the clock domain of `clk` or will it be deasserted before the clock even starts?

All these "wait, what's happening" thoughts can be neatly resolved by just using `noReset` here.

I know it's just a test bench, but the construction doesn't really make sense to me.

Reply:
Yeah, there's not going to be a delay in this level of simulation inside `ibufds`, I realise now. It's just a differential buffer.